xml/System.Text.RegularExpressions/Regex.xml

```diff
@@ -196,7 +196,7 @@
 |`\s?`|Match zero or one white-space character.|  
 |`\d*`|Match zero or more decimal digits.|  
 |`\.?`|Match zero or one decimal point symbol.|  
-|`\d{2}?`|Match two decimal digits zero or one time.|  
+|`(\d{2})?`|Match two decimal digits zero or one time.|  
 |`(\d*\.?\d{2}?){1}`|Match the pattern of integral and fractional digits separated by a decimal point symbol at least one time.|  
 |`$`|Match the end of the string.|  
   
```

Comment:
In the next sentence, "defined by the current culture's" should be changed to "defined by the en-US culture's".